README.md

```diff
@@ -577,7 +577,7 @@ Email subject to use for email change confirmation. Defaults to `Confirm Email C
 
 `MAILER_TEMPLATES_INVITE` - `string`
 
-URL path to an email template to use when inviting a user.
+URL path to an email template to use when inviting a user. (e.g. http://www.path.com/path-to-email-template.html)
```

Comment:
```suggestion
URL path to an email template to use when inviting a user. (e.g. `https://www.example.com/path-to-email-template.html`)
```

(changes being `https` should be recommended by default, and `example.com` is more sensible as an example than `path.com`, and also formats the url)